mlkem/poly.c

```diff
@@ -216,6 +241,11 @@ void mlk_poly_sub(mlk_poly *r, const mlk_poly *b)
     !MLK_USE_NATIVE_NTT && !MLK_USE_NATIVE_INTT */
 
 #if !defined(MLK_USE_NATIVE_POLY_MULCACHE_COMPUTE)
+/* Reference: Does not exist in the reference implementation.
+ *            - The reference implementation does not use a
+ *              multiplication cache ('mulcache'). This is an idea
+ *              originally taken from [eprint.iacr.org/2021/986]
```

Comment:
turn this into a proper url that's clickable https://ia.cr/2021/986

Reply:
similar for all other mentions of this paper.

Reply:
Ack